ReText/webenginepreview.py

```diff
@@ -37,12 +39,52 @@ def interceptRequest(self, info):
             info.block(True)
 
 
+class UrlPopup(QLabel):
+    def __init__(self, window):
+        super().__init__(window)
+        self.window = window
+
+        def hex_rgba(color):
+            ''' color:QColor '''
```

Comment:
Can this be made a static method or a top-level function?
This way we will have only one instance of this function for all labels.

Also, you can import QColor and use native type annotation (`color: QColor`).